src/Entity/ListBuilder/DeveloperAppListBuilderForDeveloper.php

```diff
@@ -177,16 +177,16 @@ public function render() {
   }
 
   /**
-   * Redirects users to their My apps page.
+   * Redirects users to their Apps page.
    *
    * This controller assumes that it is only invoked for authenticated users.
-   * This is enforced for the 'apigee_edge.user.my_apps' route with the
+   * This is enforced for the 'apigee_edge.user.apps' route with the
```

Comment:
I haven't found anything in the BC policy about route ids, but I guess this should not change. This chance can require a modification in any module that performed actions based on the original route id. I'd mention this name change it in the release notes at least.

Reply:
A release note should be fine.